owasp-top10-2017-apps/a7/comment-killer/killer/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.7"
+
+services:
+    hello-world:
+        container_name: comment-killer
+        build:
+            context: .
+            dockerfile: Dockerfile
+        volumes:
+            - ".:/app"
+            - "/app/node_modules"
+        ports:
+            - 3001:3000
```

Comment:
Can you use the following code into this `docker-compose.yml`, please?

```yaml
version: "3.7"

services:
  app:
    container_name: app-a7 
    build:
      context: ../
      dockerfile: deployments/Dockerfile
    ports:
      - "10007:3000"
    environment:
      - CHOKIDAR_USEPOLLING=true
    volumes:
      - ".:/app"
      - "/app/node_modules"
    restart: always
```

You will need to fix your `deployments` folder! 😊